fe/fe-core/src/main/java/com/starrocks/lake/compaction/CompactionTask.java

```diff
@@ -116,10 +112,15 @@ public void sendRequest() {
 
     public void abort() {
         if (!isCompleted()) {
-            LOG.info("abort compaction task, txn_id: {}, node: {}", request.txnId, nodeId);
             AbortCompactionRequest abortRequest = new AbortCompactionRequest();
             abortRequest.txnId = request.txnId;
-            Future<AbortCompactionResponse> ignored = rpcChannel.abortCompaction(abortRequest);
+            try {
+                Future<AbortCompactionResponse> ignored = rpcChannel.abortCompaction(abortRequest);
+                LOG.info("aborted compaction task, txn_id: {}, node: {}", request.txnId, nodeId);
+            } catch (Exception e) {
```

Comment:
this means the caller will not be aware that the `AbortCompactionRequest` is actually sent out or not, is it expected?

Reply:
Yes, it's by design.